File: 1486.py
# Author: Zoljargal Gantumur
# Runtime: 28 ms, faster than 83.37% of Python3 online submissions
# Memory Usage: 13.9 MB, less than 100.00% of Python3 online submissions

class Solution:
    def xorOperation(self, n: int, start: int) -> int:
        ans = start
        for i in range(1,n):
            ans = ans ^ (start + 2*i)
        return ans

File: 1486.cpp
/*
	Author: Zoljargal Gantumur
	Runtime: 0 ms, faster than 100.00% of C++ online submissions
	Memory Usage: 6 MB, less than 100.00% of C++ online submissions
*/

class Solution {
public:
    int xorOperation(int n, int start) {
        int ans = start;
        for(int i=1; i<n; i++){
            ans = ans ^ (start+2*i);
        }
        return ans;
    }
};
